tensorflow/lite/micro/tools/benchmarking/generic_model_benchmark.cc

```diff
@@ -45,15 +58,17 @@ namespace {
 
 using Profiler = ::tflite::MicroProfiler;
 
-constexpr int kTfLiteAbort = -9;
-
 // Seed used for the random input. Input data shouldn't affect invocation timing
 // so randomness isn't really needed.
 constexpr uint32_t kRandomSeed = 0xFB;
 
+#if !defined(MODEL_HEADER_PATH)
 constexpr size_t kTensorArenaSize = 3e6;
-constexpr int kNumResourceVariable = 100;
 constexpr size_t kModelSize = 2e6;
+#else
+constexpr size_t kTensorArenaSize = TENSOR_ARENA_SIZE;
```

Comment:
Why did you need to make the tensor arena adjustable? I'm guessing the answer is "so it could fit" in one of our targets. I'd generally like to make this as absolutely big as possible, as the tool's output will then tell you how much it really needed.

Since we know we can fit 3e6 + 2e6 in the command line version, but don't need the 2e6 model space in the builtin version, maybe we could have the `kTensorArenaSize = 5e6 - g_model_size`?

Reply:
done